README.md

```diff
@@ -51,13 +51,15 @@ assemblers:
   abyss:
     kmers: "27"
 genomagic:
+  assemblers: ['abyss','megahit','flye']
   inputFilePath: "/test/raw_sequences.fastq"
   outputPath: "/test/output"
   threads: 2
   prep: true
 ```
-
-Note: all paths used with GenoMagic have to be absolute paths (a Docker requirement).
+Note: 
+ - all paths used with GenoMagic have to be absolute paths (a Docker requirement).
+ - values for the `assemblers` field have to be lowercase as written in [constants/util.go](https://github.com/genomagic/genomagic/blob/master/constants/util.go#L21-L31).
```

Comment:
Instead of pointing users to a Go file, I would simply add the available assemblers here:
```Markdown
Available assemblers:
1. Abyss
1. Flye
1. ...
```